lib/internal/worker.js

```diff
@@ -14,17 +14,18 @@ const {
 
 const { internalBinding } = require('internal/bootstrap/loaders');
 const { MessagePort, MessageChannel } = internalBinding('messaging');
-const { handle_onclose } = internalBinding('symbols');
+const {
+  handle_onclose: handleOnCloseSymbol,
+  oninit: onInitSymbol
+} = internalBinding('symbols');
 const { clearAsyncIdStack } = require('internal/async_hooks');
 const { serializeError, deserializeError } = require('internal/error-serdes');
 const { pathToFileURL } = require('url');
 
-util.inherits(MessagePort, EventEmitter);
-
 const {
   Worker: WorkerImpl,
   getEnvMessagePort,
-  threadId
+  threadId,
```

Comment:
Nit: this triggers my OCD, can we make this consistent with the above destructuring assignment?

Reply:
Sorry, this was leftover from addressing the previous review comment  … the comma is gone now :)

Reply:
Thank you.